src/test/java/Lesson10.java

```diff
@@ -84,9 +88,10 @@ public void dynamicControl() {
     @Test
     public void uploadFileTest() {
         //open page http://the-internet.herokuapp.com/upload
-        driver.get(HEROCU_APP_UPLOAD_FILE);
+        driver.get(HEROCU_APP_URL_MAIN_PAGE + HEROCU_APP_UPLOAD_FILE);
         //find button "open file", and send absolute path to upload file
-        driver.findElement(By.id(FILE_UPLOAD_BUTTON)).sendKeys("C:\\Users\\Бубята\\IdeaProjects\\HerokuApp\\src\\test\\resources\\Lesson_4.jpg");
+        File file = new File("src/test/resources/Lesson_4.jpg");
```

Comment:
You placed jpg file in folder "target/test-classes/Lesson_4.jpg". This file should be in ""src/test/resources/Lesson_4.jpg".
I see "File not found" exception when I launch this test. Please fix it.